src/components/RadioButton/RadioButton.ts

```diff
@@ -9,30 +9,59 @@ namespace fabric {
    * Adds basic demonstration functionality to .ms-RadioButton components.
    *
    */
-  export class RadioButton extends CheckBox {
+  export class RadioButton {
```

Comment:
Why are we removing RadioButton extending CheckBox? We will have to implement the same functionality in RadioButton that are already there in CheckBox, hence prevent code reuse/DRY. RadioButton is a special type of CheckBox.